docs/technical-reference/design-contribution-workflows.md

```diff
@@ -0,0 +1,28 @@
+---
+id: design-contribution-workflows
+title: Design Contribution Workflows
+sidebar_label: Design Contribution Workflows
+--- 
+
+### Identifying design issues
+
+The OpenRefine project contains several repositories, but you’re most likely to find design-related issues in the main application repository here: https://github.com/OpenRefine/OpenRefine. Alternatively, further design issues might be linked to the main [OpenRefine website repository](https://github.com/OpenRefine/openrefine.org), or some additional extensions. 
+
+Within OpenRefine's GitHub, the [OpenRefine Design](https://github.com/orgs/OpenRefine/projects/1) Project is the main entry point for designers to identify and select suitable issues to work on.
+![This is a screenshot showing the design project in the OpenRefine GitHub repository](<../static/img/openrefine-design-project.png>)
```

Comment:
It's not really clear to me that this screenshot is useful - isn't it enough to give a link to the project (as you already did)?

Concerning the syntax to embed a URL, the simplest way to figure out the correct syntax is to look at examples from other pages in the documentation. According to those examples, you could try this:
```suggestion
![This is a screenshot showing the design project in the OpenRefine GitHub repository](/img/openrefine-design-project.png)
```

Reply:
That is brilliant. Thank you!